src/axom/quest/tests/quest_signed_distance_interface.cpp

```diff
@@ -15,19 +15,19 @@
 #include "quest_test_utilities.hpp"
 
 // gtest
-#ifdef AXOM_USE_MPI
-
-  #ifdef GTEST_HAS_DEATH_TEST
-    #undef GTEST_HAS_DEATH_TEST
-  #endif /* GTEST_HAS_DEATH_TEST */
+#include "gtest/gtest.h"
 
-  #define GTEST_HAS_DEATH_TEST 0
-#endif                    /* AXOM_USE_MPI */
-#include "gtest/gtest.h"  // for gtest macros
+// Note: Disable death tests when MPI is enabled since they're not being properly handled
+#ifdef AXOM_USE_MPI
+  #define _DEATH_TESTS_LOCALLY_DISABLED 1
+#else
+  #define _DEATH_TESTS_LOCALLY_DISABLED 0
+const char IGNORE_OUTPUT[] = ".*";
+#endif
```

Comment:
This was an odd one.

* GTest's death tests appear to be incompatible w/ MPI since they don't properly shut the app down in the spawned thread (?) leading to "Broken Pipes".
* We previously undefined GTEST_HAS_DEATH_TEST` when using MPI to avoid this
* GTest recently (~ March 2022) changed how they treat `#defines` -- they used to always be defined, w/ a value of 1 when enabled and 0 when disabled. They changed it to be defined when enabled and undefined when disabled since this allows users to use the `-Wundef` flag in their build systems.
*  This was the only file in axom that had death tests + MPI

My hack (which I'm not thrilled with) is to add another `#define` to control whether we're using the death tests, and to disable them when using MPI.